From f31d1c32075d69311242b02996dcdac4379fcd6c Mon Sep 17 00:00:00 2001
From: "Yury V. Zaytsev" <yury@shurup.com>
Date: Sun, 7 Jun 2015 20:08:43 +0200
Subject: [PATCH] Ticket #3479: Fix mc.pot comment tripping up Transifex and
 unportable inplace sed

Transifex doesn't like it when there is no space after `#` and complains
about autogenerated `/# "Plural-Forms ...` string; fix it and remove
unportable inplace sed invocation.

Signed-off-by: Yury V. Zaytsev <yury@shurup.com>
---
 po/Rules-pot-defaults |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/po/Rules-pot-defaults b/po/Rules-pot-defaults
index 80b9e55..2cf834e 100644
--- a/po/Rules-pot-defaults
+++ b/po/Rules-pot-defaults
@@ -4,10 +4,10 @@ update-po: Makefile
 	sed \
 	    -e '/#, fuzzy/d' \
 	    -e 's/Project-Id-Version: PACKAGE VERSION/Project-Id-Version: $(PACKAGE) $(VERSION)/' \
-	    -e 's/^"Plural-Forms: nplurals=INTEGER;/#"Plural-Forms: nplurals=INTEGER;/' \
+	    -e 's/^"Plural-Forms: nplurals=INTEGER;/# "Plural-Forms: nplurals=INTEGER;/' \
 	    -e 's@Content-Type: text/plain; charset=CHARSET@Content-Type: text/plain; charset=utf-8@' \
 	    < $(srcdir)/$(DOMAIN).pot > $(DOMAIN).1po && \
 	mv -f $(DOMAIN).1po $(srcdir)/$(DOMAIN).pot
 	test -z "$(UPDATEPOFILES)" || $(MAKE) $(UPDATEPOFILES)
 	$(MAKE) update-gmo && \
-	for i in $(srcdir)/*.po; do sed -i '/^#\:/d' $$i; done
+	for i in $(srcdir)/*.po; do sed '/^#\:/d' < $$i > $$i.clean && mv -f $$i.clean $$i; done
-- 
1.7.10

